packages/headless/src/features/question-answering/question-answering-slice.ts

```diff
@@ -123,9 +125,7 @@ export const questionAnsweringReducer = createReducer(
         ) => {
           const idx = findRelatedQuestionIdx(
             state.relatedQuestions,
-            action.payload as
-              | QuestionAnsweringUniqueIdentifierActionCreatorPayload
-              | QuestionAnsweringDocumentIdActionCreatorPayload
```

Comment:
The `as` is no longer necessary, because I added the missing generic type to `validateQuestionAnsweringActionCreatorPayload`